cmd/zed/zed.d/all-debug.sh

```diff
@@ -5,7 +5,9 @@
 # This can be a useful aid when developing/debugging ZEDLETs since it shows the
 # environment variables defined for each zevent.
 
+#shellcheck source=/dev/null
```

Comment:
Let's just exclude these errors for now by adding `--exclude=SC1090` to the command line.